ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -81,6 +81,12 @@ literals:
 - chilled_water_bypass_valve_status:
   - OPEN
   - CLOSED
+- conditioning_water_differential_pressure_sensor:
```

Comment:
Conditioning water is too vague as this could refer to chemically treated water, or any other treatment of water. I would recommend changing these to something like `thermally_conditioned_water_differential_pressure_sensor` and add `thermally` as a subfield.